utilities/ride-model-util/src/main/java/com/adobe/ride/utilities/model/ModelObject.java

```diff
@@ -739,10 +739,7 @@ private JSONObject buildObjectNode(String pathToParent, JSONObject obj) {
         if (existingValue == null) {
           Object genValue = null;
           try {
-            if (currentkey.equals("otherArticles")) {
-              System.out.println("Debug");
-            }
-            genValue = generateNodeValue(null, currentkey, propertyDef);
+             genValue = generateNodeValue(null, currentkey, propertyDef);
```

Comment:
Looks good!  One more change though.  The formatting is a little off (notice how genValue is a little indented).   

In general, we use the GoogleStyle java format.  If you install it into your IDE, it will help with future commits: https://github.com/google/google-java-format.

If you have trouble installing the style into your IDE, please just remove one space from before genValue in your PR.   The style formatter is preferable (tabs versus spaces, etc).  This is not a huge deal, but if you make more involved changes later, having the style formatter in place will be handy.

Reply:
I'll try and find time to update the readme with this info so that others don't run into this.  Sorry the info wasn't available to you! :(

Reply:
No problem @tedcasey indentation corrected.